app/views/articles/_image.html.erb

```diff
@@ -1,7 +1,12 @@
-<% if article.image.present? %>
-  <div class="article-image">
-    <%= link_to_if linked,
-                   image_tag(article.image, class: "u-photo", alt: article.image_description),
-                   article.path %>
-  </div>
-<% end %>
+<div class="article-image">
+  <img class="u-photo"
+       alt="<%= article.image_description %>"
+       src="<%= article.article_image.url || article.image %>"
+       <% if article.article_image.url.present? %>
+         sizes="(min-width: 36em) 33.3vw,100vw"
+         srcset="<%= article.article_image.large %>  1024w,
+                 <%= article.article_image.medium %> 640w,
+                 <%= article.article_image.small %>  320w"
```

Comment:
See above about headers being 2000px at their big size.

Also, I'm not up on the `sizes` and `srcset` attrs. Can you explain to me in English what is happening here?